kimera_vio_ros/src/stereo_vio_node.cpp

```diff
@@ -0,0 +1,23 @@
+#include "gflags/gflags.h"
+#include "glog/logging.h"
+#include "kimera_vio_ros/components/stereo_vio.hpp"
+
+using StereoVio = kimera_vio_ros::components::StereoVio;
+
+int main(int argc, char * argv[])
+{
+  auto g_args = rclcpp::init_and_remove_ros_arguments(argc, argv);
+  int g_argc = g_args.size();
+  // Initialize Google's flags library.
+  google::ParseCommandLineFlags(&g_argc, &argv, true);
```

Comment:
@ToniRV , when using composed nodes, I don't think we'd be able to acquire the CLI args for the containing process that holds all the nodes. Thus we'll need a way to pass any current run time args for google flag/log using ROS parameters instead. Perhaps we could a ROS parameter for passing a directories path containing google flag files, like with the param path for yaml files. Or would there be would be a way to combine the two?